Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -232,6 +232,9 @@ public class GraphTraverser: GraphTraversing {
 
     public func searchablePathDependencies(path: AbsolutePath, name: String) throws -> Set<GraphDependencyReference> {
         try linkableDependencies(path: path, name: name, shouldExcludeHostAppDependencies: false)
+            .union(
+                staticPrecompiledDependenciesAndTheirDependencies(path: path, name: name)
+            )
```

Comment:
nit
```suggestion
            .union(staticPrecompiledDependenciesAndTheirDependencies(path: path, name: name))
```

Reply:
applied in https://github.com/tuist/tuist/pull/5107/commits/61f6762d26191e6b0db2c7f8c4bf350784b47b0a